hack/test

```diff
@@ -37,7 +39,15 @@ if [ "$TEST_COVERAGE" = "1" ]; then
   export GO_TEST_COVERPROFILE="/testreports/coverage-report$TEST_REPORT_SUFFIX.txt"
 fi
 
-cid=$(docker create --rm --privileged \
+dockerConfigMount=""
+if [ "$GITHUB_ACTIONS" = "true" ]; then
+  dockerConfigPath="$HOME/.docker/config.json"
+  if [ -f "$dockerConfigPath" ]; then
+    dockerConfigMount="-v $dockerConfigPath:/root/.docker/config.json:ro"
```

Comment:
Would it make sense to use the `--mount` syntax for this, instead of the `-v` shorthand? (not sure if quoting is needed);


```bash
--mount type=bind,"src=$dockerConfigPath",dst=:/root/.docker/config.json,readonly
```